trojstenid/locale/sk/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,1896 @@
+msgid "Login"
+msgstr "Prihlásiť sa"
+
+#: socialaccount/providers/dummy/templates/dummy/authenticate_form.html:19
+#: templates/account/base_confirm_code.html:48
+#: templates/account/base_confirm_code.html:52
+#: templates/account/password_reset_from_key.html:33
+#: templates/idp/oidc/authorization_form.html:26
+#: templates/mfa/authenticate.html:28 templates/mfa/authenticate.html:44
+#: templates/mfa/trust.html:29 templates/mfa/webauthn/signup_form.html:26
+msgid "Cancel"
+msgstr "Zrušiť"
+
+#: socialaccount/providers/oauth/client.py:84
+#, python-format
+msgid ""
+"Invalid response while obtaining request token from \"%s\". Response was: %s."
+msgstr "Neplatná odpoveď pri získavaní request tokenu z \"%s\". Odpoveď: %s."
+
+#: socialaccount/providers/oauth/client.py:118
+#: socialaccount/providers/pocket/client.py:86
+#, python-format
+msgid "Invalid response while obtaining access token from \"%s\"."
+msgstr "Neplatná odpoveď pri získavaní prístupového tokenu z \"%s\"."
+
+#: socialaccount/providers/oauth/client.py:139
+#, python-format
+msgid "No request token saved for \"%s\"."
+msgstr "Žiadna uložená požiadavka tokenu pre \"%s\"."
+
+#: socialaccount/providers/oauth/client.py:190
+#, python-format
+msgid "No access token saved for \"%s\"."
+msgstr "Žiadny uložený prístupový token pre \"%s\"."
+
+#: socialaccount/providers/oauth/client.py:211
+#, python-format
+msgid "No access to private resources at \"%s\"."
+msgstr "Žiadny prístup do privátneho úložiska na \"%s\"."
+
+#: socialaccount/providers/pocket/client.py:41
+#, python-format
+msgid "Invalid response while obtaining request token from \"%s\"."
+msgstr "Neplatná odpoveď pri získavaní request tokenu z \"%s\"."
+
+#: templates/account/account_inactive.html:5
+#: templates/account/account_inactive.html:9
+msgid "Account Inactive"
+msgstr "Účet neaktívny"
+
+#: templates/account/account_inactive.html:12
+msgid "This account is inactive."
+msgstr "Tento účet je neaktívny."
+
+#: templates/account/base_confirm_code.html:27
+#, python-format
+msgid ""
+"We've sent a code to %(recipient)s. The code expires shortly, so please "
+"enter it soon."
+msgstr ""
+"Odoslali sme overovací kód na e-mail %(recipient)s. Platnosť tohto kódu "
+"čoskoro vyprší, zadaj ho čo najskôr."
+
+#: templates/account/base_confirm_code.html:39
+#: templates/account/email_confirm.html:24
+#: templates/account/reauthenticate.html:18
+#: templates/idp/oidc/device_authorization_confirm_form.html:26
+#: templates/mfa/reauthenticate.html:18
+msgid "Confirm"
+msgstr "Potvrdiť"
+
+#: templates/account/base_confirm_code.html:43
+msgid "Request new code"
+msgstr "Vyžiadať nový kód"
+
+#: templates/account/base_reauthenticate.html:5
+#: templates/account/base_reauthenticate.html:9
+msgid "Confirm Access"
+msgstr "Potvrdiť prístup"
+
+#: templates/account/base_reauthenticate.html:12
+msgid "Please reauthenticate to safeguard your account."
+msgstr "Znovu sa prihlás pre zvýšenú bezpečnosť."
```

Comment:
Toto mi príde, že sa to dá sformulovať lepšie, ale nenapadá mi momentálne nič lepšie